ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/monitor-layer1.j2

```diff
@@ -7,6 +7,9 @@
 {% if monitor_layer1.enabled is arista.avd.defined(true) %}
 !
 monitor layer1
+{%     if monitor_layer1.logging_transceiver.dom_and_communication is arista.avd.defined(true) %}
```

Comment:
please add a molecule for the same

Reply:
Added molecule test with name monitor-layer1-a.yml